.github/workflows/bindings.yml

```
@@ -13,7 +13,7 @@ env:
   CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
   CIBW_BUILD: cp3?-*
```

```suggestion
  CIBW_BUILD: cp3*
```
Don't you want to add support for Python 3.10?